### PR TITLE
Add engrampa support - closes #356

### LINKED ISSUE #356: Create new extension to integrate with engrampa
Nemo has the nemo-fileroller to integrate with Gnome's file roller and a similar extension should be created to integration with Engrampa, Mate's file manager.

### PR DESCRIPTION
**Pull request by fossfreedom**
This PR adds a new extension targeted at using engrampa rather than file-roller.

This has been developed on ubuntu budgie 22.04 without file-roller/nemo-fileroller installed.

The same two right click menu options are available and functional as for nemo-fileroller.

----

As an alternative solution for a new extension, maybe nemo-fileroller should be renamed "nemo-archiver" - and depending upon a meson_option being used either fileroller or engrampa can be compiled into the code. If this is the preferred method then I'm happy to-do.

### CODE REVIEW

**Comment by ItzSwirlz** [comment]
This is cool; would it make more sense (since this is a small extension) to just make one extension that supports both file-roller and engrampa instead?

**Comment by mtwebster** [comment]
I think I'd rather go with the second option.

Is it imperative the package be renamed? We try to avoid it as you end up having to use tedious things (at least to us) like replaces/conflicts, transitional packages, etc...

dpkg-gencontrol could be used to fix the fileroller/engrampa dependency as well as alter the binary package name. Maybe it can modify the description as well, but worst case you can reword it to be a bit more generic if necessary.

Also, something to consider: you could use NemoActions to entirely replace this extension. At a glance the only thing it won't do currently is exclude certain uri, though you can require one (like file). Check out the great [documentation](https://github.com/linuxmint/nemo/blob/master/files/usr/share/nemo/actions/sample.nemo_action).
